python_modules/automation/automation/docker/images/buildkite-test/Dockerfile

```diff
@@ -169,7 +168,7 @@ COPY --from=snapshot_builder /snapshot-reqs.txt .
 
 # Preinstall non-Dagster packages in image so that the virtual environment
 # builds faster in Buildkite.
-RUN uv pip install -r /snapshot-reqs.txt \
+RUN uv pip install --system -r /snapshot-reqs.txt \
```

Comment:
why is this change necessary?

Reply:
Unlike pip, uv doesn't allow installation without a virtual environment by default.

It should still be possible to `pip install` (instead of `uv pip install`), but imagine that would be unnecessarily slower, especially if everything else is using uv.